File: Java/1180.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
		Scanner leitor = new Scanner(System.in);	

		int n = leitor.nextInt();
		int menor = 0, pos = 0;
		int x[] = new int[n];
		
		for (int i = 0; i < n; i++){
			x[i] = leitor.nextInt();
		}
		
		for (int j = 0; j < n; j++){
			if (x[j] < menor){
				menor = x[j];
				pos = j;
			}
		}
    	System.out.printf("Menor valor: %d\n", menor);
    	System.out.printf("Posicao: %d\n", pos);
    }
}

File: Python/1180.py
# -*- coding: utf-8 -*-
qntd = int(input())
lista = list(map(int,input().split()))

posicao = 0
menor = lista[0]
cont = 0
for v in lista:
    if(v < menor):
        menor = v
        posicao = cont
    cont += 1
    
print("Menor valor: %d" % menor)
print("Posicao: %d" % posicao)